draft/0004-user-interaction.md

```diff
@@ -0,0 +1,497 @@
+
+| Interaction Session State | Description                                                                                    | Restrictions                                                                                                                                              |
+| ------------------------- | ---------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `initiated`               | The interaction session is initiated when a user sends an authentication request to the Logto. | The user can only have one active interaction session at a time. All experience APIs require a valid interaction session.                                 |
+| `identified`              | The user has provided a valid identifier and at least one related verification record.         | The user can be identified by the Logto system. The interaction session can access the user's profile and MFA settings for further security checks.       |
+| `verified`                | The user has provide all necessary verification records.                                       | The user is fully verified by the Logto system. The user can proceed to the user profile completion and authentication process based on the SIE settings. |
+| `authenticated`           | The interaction session has been submitted. Authorization granted                              | The user can access the platform resources and services. The interaction session will be terminated once the user logs out or the session expires.        |
+
+### 5.2 Interaction session data
+
+| Field               | Type             | Description                                                                                                                                    |
+| ------------------- | ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| interactionEvent    | InteractionEvent | he interaction event for the current interaction session.                                                                                      |
+| accountId           | string           | The accountId of the user for the current interaction session. Only available once the user is identified by at least one verified identifier. |
+| verificationRecords | Verification[]   | The verification record list attached to the current interaction session.                                                                      |
+| profile             | Profile          | The user provided profile data in the current interaction session that needs to be stored to user DB.                                          |
+
+## Appendix
+
+### A. Experience APIs
```

Comment:
the header level is too deep (saw h5 below). we should use h4 as the highest level. specifically in this case, will it be better to use:

```md
## Appendix A. Experience API
## Appendix B. Something else
```